proseco/tests/test_catalog.py

```diff
@@ -357,7 +357,8 @@ def test_t_ccd_effective_acq_guide(t_ccd_case):
 
 
 def test_call_args_attr():
-    aca = get_aca_catalog(**mod_std_info(optimize=False, n_guide=0, n_acq=0, n_fid=0))
+    aca = get_aca_catalog(**mod_std_info(optimize=False, n_guide=0, n_acq=0, n_fid=0),
+                          raise_exc=False)
```

Comment:
Not sure about this one.

Reply:
This is reproducing the original default, and this test requires `raise_exc` to pass.

Reply:
I meant it needs `raise_exc=False` to pass.

Reply:
I suppose it really just needs raise_exc to *do* the test on call_args.